apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -122,7 +122,16 @@ public static IApplicationBuilder UseGoogleTrace(this IApplicationBuilder app)
             services.TryAddSingleton<IHttpContextAccessor, HttpContextAccessor>();
 
             services.AddSingleton(ManagedTracer.CreateDelegatingTracer(() => ContextTracerManager.GetCurrentTracer()));
+
+#if NETSTANDARD2_0
+            // On .Net Standard 2.0 or higher, we can use the System.Net.Http.IHttpClientFactory defined in Microsoft.Extensions.Http,
+            // for which we need a DelagatingHandler with no InnerHandler set. This is the recommended way.
+            // It should be registered as follows.
+            services.AddTransient(sp => new UnchainedTraceHeaderPropagatingHandler(() => ContextTracerManager.GetCurrentTracer()));
```

Comment:
Does `new UnchainedTraceHeaderPropagatingHandler(ContextTracerManager.GetCurrentTracer)` work? (Just replacing the lambda expression with a method group conversion.)

Possibly likewise for TraceHeaderPropagatingHandler.

Reply:
Oh yes, it should work, I just copy pasted from the existing one. Will change.